dev/artifacts/canonical.manifest

```diff
@@ -7,8 +7,10 @@
 # http://polymer.github.io/PATENTS.txt
 
 # this is a list of recipes that are used in our canonical demos.
-import 'Profile/Profile.recipes'
 import 'Events/Events.recipes'
+import 'People/People.recipes'
+import 'Places/Places.recipes'
+import 'Profile/Profile.recipes'
 import 'Restaurants/Restaurants.recipes'
 
 import 'Products/Products.recipes'
```

Comment:
do we want to keep the alphabetized? or group by test cases? if the latter, probably comments will be helpful. otoh it might be confusing as we have more tests.